wp-content/themes/workingnyc/search.php

```diff
@@ -14,17 +14,14 @@
 // Create query
 $wp_query = new WP_Query(array(
```

Comment:
I added the post_type filter to WP_Query so that it would automatically only include jobs and programs in search results. That way, I could remove the code below to filter search results afterwards